frontend/src/lib/components/TimezoneAware/index.tsx

```diff
@@ -90,13 +99,22 @@ export function TZLabel({ time, showSeconds }: { time: string | dayjs.Dayjs; sho
 /** Return an explainer component for analytics visualization pages. */
 export function TZIndicator({ style }: { style?: React.CSSProperties }): JSX.Element {
     const { currentTeam } = useValues(teamLogic)
+
+    const { reportTimezoneComponentViewed } = useActions(eventUsageLogic)
+
+    const handleVisibleChange = (visible: boolean): void => {
+        if (visible) {
+            reportTimezoneComponentViewed('indicator', currentTeam?.timezone, shortTimeZone())
+        }
+    }
+
     const PopoverContent = (
         <div className="tz-label-popover">
             <TZConversionHeader />
             <p style={{ maxWidth: 320 }}>
-                Times presented in visualizations are UTC.
+                All graphs are calculated and presented in UTC.
```

Comment:
UTC -> GMT ?

Reply:
also maybe "GMT time"? 

Even if 'T' stands for time maybe this makes it a bit clearer?